packages/visual-editor/src/components/VisualEditorRoot.tsx

```diff
@@ -7,11 +7,27 @@ import { OUTGOING_EVENTS } from '@contentful/experience-builder-core/constants';
 import { useInitializeEditor } from '@/hooks/useInitializeEditor';
 import { useEntityStore } from '@/store/entityStore';
 import { useEditorStore } from '@/store/editor';
+import { useZoneStore } from '@/store/zone';
+import { CTFL_ZONE_ID } from '@/types/constants';
+
+const findNearestDropzone = (element: HTMLElement) => {
+  const zoneId = element.getAttribute(CTFL_ZONE_ID);
+
+  if (!element.parentElement) {
+    return '';
+  }
+
+  if (element.tagName === 'BODY') {
+    return '';
+  }
```

Comment:
What do you think about `return null` instead of `return ''` on these early returns?  This may be a nit pick but `null` seems more appropriate for representing the absence of a value

Reply:
Yeah good call here